index.html

```diff
@@ -841,6 +841,65 @@ <h3 class="r-fit-text" style="margin-top: 8rem;">
                     </section>
                 </section>
 
+                <!-- Honourable mention -->
+                <section>
+                    <section data-background-image="images/TyPower.svg"
+                             data-background-size="15%" data-background-position="right">
+                        <h2>Please also have a look at the following teams</h2>
+                        <ul>
+                            <li>
+                                <a href="/community/teams/typo3-development" title="TYPO3 Development">TYPO3 Development</a>
```

Comment:
The links need to be absolute links to t30 to work (as the slides usually are run from a local git clone).

Also, if the link texts are the same as the link titles, we don't need the titles.

Reply:
I set the h2 tag on separate lines.